src/main/resources/selogerLastPage.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<recherche>
+<resume>ventes75015 triées par prix.</resume>
+<resumeSansTri>ventes75015</resumeSansTri>
+<nbTrouvees>1432</nbTrouvees>
+<nbAffichables>200</nbAffichables>
+<pageCourante>4</pageCourante>
+<pageMax>4</pageMax>
+<pagePrecedente>http://ws.seloger.com/http://ws.seloger.com/search.xml?cp=75015&amp;idtt=2&amp;SEARCHpg=3</pagePrecedente>

```

Comment:
why do we need this resource? If it's for a test, it should be in `src/test/resources`